blockly-ws-search/workspace_search.js

```diff
@@ -108,16 +108,18 @@ class WorkspaceSearch {
     // TODO: Figure out how we are going to deal with translating.
     textInput.setAttribute('placeholder', 'Search');
     this.textInput_ = textInput;
-    Blockly.bindEventWithChecks_(textInput, 'keydown', this, this.onKeyDown_);
-    Blockly.bindEventWithChecks_(textInput, 'input', this, this.onInput_);
-    Blockly.bindEventWithChecks_(svg.parentNode, 'keydown', this,
-        this.onWorkspaceKeyDown_);
+    textInput.addEventListener('keydown', this.onKeyDown_.bind(this));
+    textInput.addEventListener('input', this.onInput_.bind(this));
+    svg.parentNode.addEventListener('keydown',
+        this.onWorkspaceKeyDown_.bind(this));
```

Comment:
Recommend using arrow function to deal with binding this. See lower review comment for more details.